package/src/components/Button/Button.test.js

```diff
@@ -1,12 +1,25 @@
 import React from "react";
-import renderer from "react-test-renderer";
-import mockComponents from "../../tests/mockComponents";
+import { cleanup, render } from "../../tests/index.js";
 import Button from "./Button";
 
-test("basic snapshot", () => {
-  const component = renderer.create(<Button components={mockComponents} title="title" className="a b">Submit</Button>);
+afterEach(cleanup);
 
-  const tree = component.toJSON();
-  expect(tree).toMatchSnapshot();
+test("basic snapshot - only default props", () => {
+  const { asFragment } = render(<Button className="myBtn">Submit</Button>);
+  expect(asFragment()).toMatchSnapshot();
 });
 
+test("error button snapshot", () => {
+  const { asFragment } = render(<Button className="myBtn" color="error" variant="contained">Delete</Button>);
+  expect(asFragment()).toMatchSnapshot();
+});
+
+test("error button snapshot", () => {
+  const { asFragment } = render(<Button className="myBtn" color="error" variant="outlined">Delete</Button>);
+  expect(asFragment()).toMatchSnapshot();
+});
+
+test("isWaiting button snapshot", () => {
```

Comment:
I believe you could also do a jest-dom [toBeDisabled](https://github.com/testing-library/jest-dom#tobedisabled) assertion here.